<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectralScanData_Dlg</class>
 <widget class="QDialog" name="SpectralScanData_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>751</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>检测结果</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>731</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Quit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Display">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Del">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Dis">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>回放</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>Btn_Display</tabstop>
  <tabstop>Btn_Del</tabstop>
  <tabstop>Btn_Dis</tabstop>
  <tabstop>Btn_Quit</tabstop>
  <tabstop>dateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
